docs/content/en/docs/whatsnew/changelog.md

```diff
@@ -15,7 +15,49 @@ Please upgrade to `v0.15.3` if you are using Tinkerbell (Bare Metal) provider.
 * When upgrading to a new minor version, a new OS image must be created using the new image-builder cli.
 {{% /alert %}}
 
-[Unreleased]
+## [v0.18.0](https://github.com/aws/eks-anywhere/releases/tag/v0.18.0)
+
+### Supported OS version details
+|              | vSphere | Bare Metal |  Nutanix | CloudStack | Snow  |
+|    :---:     |  :---:  |   :---:   |   :---:  |    :---:   | :---: |
+| Ubuntu       | 20.04	 | 20.04     |	20.04             | Not supported	     | 20.04 |  
+|              | 22.04   | 22.04     |  22.04             | Not supported      | Not supported   |
+| Bottlerocket | 1.14.3  | 1.14.3    |  Not supported     | Not supported	     | Not supported   |
+| RHEL         | 8.7	   | 8.7	     |  Not supported     | 8.7	               | Not supported   |
+
+
+### Added
+- Etcd encryption for CloudStack and vSphere: [#6557](https://github.com/aws/eks-anywhere/pull/6557)
+- KinD-less upgrades: [#6622](https://github.com/aws/eks-anywhere/pull/6622)
+- Generate TinkerbellTemplateConfig command: [#3588](https://github.com/aws/eks-anywhere/issues/3588)
+- Support for modular Kubernetes version upgrades with bare metal: [#6735](https://github.com/aws/eks-anywhere/pull/6735)
+  - OSImageURL added to Tinkerbell Machine Config
+- Bare metal out-of-band webhook: [#5738](https://github.com/aws/eks-anywhere/issues/5738)
+- Support for Kubernetes v1.28
+- Support for air gapped image building: [#6457](https://github.com/aws/eks-anywhere/issues/6457)
+
```

Comment:
- Support for RHEL 8 and RHEL 9 for Nutanix provider: [#6822](https://github.com/aws/eks-anywhere/pull/6822)